flake.nix

```diff
@@ -14,15 +14,35 @@
       url = "github:numtide/devshell";
       inputs.nixpkgs.follows = "nixpkgs";
     };
+    flake-compat = {
+      url = "github:input-output-hk/flake-compat/fixes";
+      flake = false;
+    };
+    blank.url = "github:input-output-hk/empty-flake";
     process-compose.url = "github:Platonic-Systems/process-compose-flake";
     services-flake.url = "github:tgunnoe/services-flake";
 
     # Sidechains deps
-    trustless-sidechain.url = "github:input-output-hk/trustless-sidechain/v6.0.0-rc3";
-    cardano-node.url = "github:IntersectMBO/cardano-node/1.35.7";
-    cardano-dbsync.url = "github:IntersectMBO/cardano-db-sync/13.1.1.3";
-    ogmios.url = "github:mlabs-haskell/ogmios";
-    kupo.url = "github:mlabs-haskell/kupo-nixos";
+    trustless-sidechain = {
```

Comment:
SHouldn't we rename this to partner chains smart contracts?

Reply:
yeah, i was just afraid i'd be accused of doing too much at the same time with this PR